network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_author_detail_header.html

```diff
@@ -0,0 +1,49 @@
+{% load l10n i18n localization static blog_tags wagtailimages_tags wagtailroutablepage_tags wagtailcore_tags %}
+
+{% get_root_or_page as blog_index_page %}
+
+<div class="tw-grid tw-grid-cols-12 large:tw-gap-40">
+    <div class="tw-col-span-12 large:tw-col-span-8">
+        <div class="tw-text-sm tw-text-gray-40">
+            <a class="tw-text-gray-40" href="{% pageurl blog_index_page %}">Blog</a>
+            <span> › </span>
+        </div>
+        <div class="tw-gap-8 tw-pt-10 medium:tw-inline-flex medium:tw-pt-7">
+            <div id="image-container" class="medium:tw-order-last">
+                <div class="tw-flex-shrink-0">
+                    {% with profile_image_classes="tw-rounded-full tw-w-28 tw-h-28" %}
+                        {% if author.image %}
+                            {% image author.image fill-120x120 as img_1x %}
+                            {% image author.image fill-240x240 as img_2x %}
+                            <img srcset="{{ img_1x.url }}, {{ img_2x.url }} 2x" src="{{ img_2x.url }}" width="{{ img_2x.width|unlocalize }}" height="{{ img_2x.height|unlocalize }}"  alt="{{ img_2x.alt }}" class="{{ profile_image_classes }}" />
+                        {% else %}
+                            <img src="{% static '_images/author-placeholder.jpg' %}" width="58px" height="58px" alt="{{ author.name }}" class="{{ profile_image_classes }}" />
+                        {% endif %}
+                    {% endwith %}
+                </div>
+            </div>
+            <h1 class="tw-mb-8">{{ author.name }}</h1>
+        </div>
+        <p class="tw-text-xl tw-pb-16">
+            {{ author.localized.introduction }}
```

Comment:
We should avoid using `localized` in the templates, as it generates extra queries. I think `author` should already be localized, though I'm not sure. This would be a good scenario to add tests.

Reply:
Updated! Working on adding tests now 👍

Reply:
Hey @jhonatan-lopes, after taking a further look at this, I agree that the `.localized` is no longer needed here.

That is because this page is using profile slugs to grab/render the data needed from the DB, and each translation of a profile has its own unique slug (ex: EN would be `daniel-miranda-1` and FR would be `daniel-miranda-4`).

That means if a user visits the blog author index page under their preferred locale and selects my name, they will be redirected to my author detail page with the appropriate slug if applicable.( if not, it will default to the EN slug). 

One concern that I have, is that this means that if a user visits `/fr/blog/authors/daniel-miranda-1` (the English slug), that the profile details will still render in English. 

However, I know that we have implemented profile slugs like this [by design](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/10578), so I think that this is OK for now, and we can discuss this issue further at a later time.

I have added it to the agenda on the next sprintly engineering meeting, and will add some simple tests making sure that the pages of translated profiles render the appropriate language. 

Thanks!